simulate_groups.py

```diff
@@ -78,7 +81,16 @@ def simulate_ll(n,
                 # all variables in same group have same coefficient
                 # (0 if we drop out that group)
                 # draw from N(0, 1)
-                B[group] = np.random.randn()
+                if prev_betas is None:
+                    B[group] = np.random.randn()
+                # if there's a previous value of beta, add noise and
+                # confirm same directionality of old and new beta
+                else:
+                    new_beta = 0
+                    while new_beta * prev_betas[group][0] <= 0:
+                        added_noise = np.random.normal(scale=0.3)
```

Comment:
Maybe you could make the scale here a parameter to the function, or make it a constant at the top of the file (e.g. `BETA_NOISE_SCALE = 0.3` after the imports)?

Reply:
Good call.